File: Entidades/NotaVentaEN.vb

Public Class NotaVentaEN
    Private _codnot As Integer
    Public Property CodNot() As Integer
        Get
            Return _codnot
        End Get
        Set(ByVal value As Integer)
            _codnot = value
        End Set
    End Property

    Private _nronota As String
    Public Property NroNota() As String
        Get
            Return _nronota
        End Get
        Set(ByVal value As String)
            _nronota = value
        End Set
    End Property

    Private _codcli As Integer
    Public Property CodCli() As Integer
        Get
            Return _codcli
        End Get
        Set(ByVal value As Integer)
            _codcli = value
        End Set
    End Property


    Private _fecha As String
    Public Property Fecha() As String
        Get
            Return _fecha
        End Get
        Set(ByVal value As String)
            _fecha = value
        End Set
    End Property

    Private _activo As String
    Public Property Activo() As String
        Get
            Return _activo
        End Get
        Set(ByVal value As String)
            _activo = value
        End Set
    End Property

    Private _detalle As List(Of DetalleEN)
    Public Property Detalle() As List(Of DetalleEN)
        Get
            Return _detalle
        End Get
        Set(ByVal value As List(Of DetalleEN))
            _detalle = value
        End Set
    End Property


End Class

File: Entidades/NotaVentaEN.cs
using System.Collections.Generic;

namespace Entidades
{
    public class NotaVentaEN
    {
        private int _codnot;

        public int CodNot
        {
            get
            {
                return _codnot;
            }

            set
            {
                _codnot = value;
            }
        }

        private string _nronota;

        public string NroNota
        {
            get
            {
                return _nronota;
            }

            set
            {
                _nronota = value;
            }
        }

        private int _codcli;

        public int CodCli
        {
            get
            {
                return _codcli;
            }

            set
            {
                _codcli = value;
            }
        }

        private string _fecha;

        public string Fecha
        {
            get
            {
                return _fecha;
            }

            set
            {
                _fecha = value;
            }
        }

        private string _activo;

        public string Activo
        {
            get
            {
                return _activo;
            }

            set
            {
                _activo = value;
            }
        }

        private List<DetalleEN> _detalle;

        public List<DetalleEN> Detalle
        {
            get
            {
                return _detalle;
            }

            set
            {
                _detalle = value;
            }
        }
    }
}